src/ccalltest.c

```diff
@@ -26,7 +27,7 @@ int __declspec(noinline)
 #else
 int __attribute__((noinline))
 #endif
-testUcharX(unsigned char x) {
+DLLEXPORT testUcharX(unsigned char x) {
```

Comment:
for other reviewers: this change shouldn't be necessary, since dllexport is the default for Mac, Linux, and Mingw64 compilers. However, it could perhaps be somewhat useful if dealing with MSVC in the future.